src/parentheses/Solution.java

```diff
@@ -0,0 +1,34 @@
+package parentheses;
+
+import java.util.Stack;
+
+public class Solution {
+    public boolean isValid(String parentheses) {
+
+        char[] arrayParentheses = parentheses.toCharArray();            // O(parentheses)
+                                                                        // time          Space
+        Stack<Character> stack = new Stack<>(); // O(1)
+        boolean result = true;
+        for (int i = 0; i < arrayParentheses.length; i++) {            // O(arrayParentheses.length);  O(1)
+            char character = arrayParentheses[i];                      // O(1);         O(1)
+
+            if (character == '[' ||                                    // O(1)         O(0)
+                    character == '{' ||
+                    character == '(') {
+                stack.push(character);                                  // O(1)         O(1)
+                System.out.println("stack: " + stack);
+            } else {
+                Evaluation answer = new Evaluation();
```

Comment:
Some questions:
1. Why do we need to create a new class?
2. What is the advantage of it?
3. When should we create a class and when we should not?